source/blog/2018-12-21-the-ember-times-issue-78.md

```diff
@@ -12,7 +12,27 @@ responsive: true
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [What's New in Ember Inspector and Extensible Ember Inspector 🐹](https://github.com/emberjs/rfcs/pull/417)
+
+We've been working hard over the past few months to close out lots of
+super old issues (some from several years ago!), and work on adding new
```

Comment:
Maybe add a link to one of the really old issues that was closed? Might be nice. Not a must.

Reply:
Also maybe this:
`and work on adding new`
Could be changed to:
`as well as adding new`
The current grammar is a bit wacky :)